hic_qc.py

```diff
@@ -628,39 +724,19 @@ def pass_judgement(self):
             self.judge_bad (bool): does the hi-c library show 'bad' characteristics, e.g. zero-distance reads or too many duplicates.
             self.judge_html (str): an HTML string to put into pass/fail box
         '''
-        if self.stats['total_read_pairs_hq'] > 0:
-            long_contacts = self.stats['pairs_intracontig_hq_gt10kbp'] / self.stats['total_read_pairs_hq'] > 0.01
-            long_floor = self.stats['pairs_intracontig_hq_gt10kbp'] / self.stats['total_read_pairs_hq'] > 0.01
-            useful_contacts = self.stats['intercontig_pairs_hq'] / self.stats['total_read_pairs_hq'] > 0.1
-            many_zero_pairs = self.stats['zero_dist_pairs'] / self.stats['total_read_pairs'] > 0.4
-            many_many_zero_pairs = self.stats['zero_dist_pairs'] / self.stats['total_read_pairs'] > 0.2
-            high_dupe = (self.stats['duplicate_reads'] / self.stats['total_reads'] > 0.1 and self.stats['total_read_pairs'] <= 1e6) \
-                        or (self.stats['duplicate_reads'] / self.stats['total_reads'] > 0.3 and self.stats['total_read_pairs'] <= 1e8)
-        else:
-            long_contacts = False
-            long_floor = False
-            useful_contacts = False
-            many_zero_pairs = False
-            many_many_zero_pairs = False
-            high_dupe = False
-
-        low_contiguity = self.N50 < 100000
+        # We are stricter on wanting a low number of dupes when it looks like we are only looking at a QC amount of sequencing (<10M read pairs)
+        allowed_dupe_percentage = 1.0 if self.stats['total_read_pairs'] > 1e7 else 0.5
```

Comment:
It might be good to set this as a property since it's calculated by multiple functions

Reply:
Good idea. Did that